inlineplz/linters/bandit.py

```diff
@@ -3,10 +3,35 @@
 from __future__ import unicode_literals
 
 import dirtyjson as json
+import sys
```

Comment:
[inline-plz]: `prospector: pylint: standard import "import sys" should be placed before "import dirtyjson as json" (wrong-import-order)`